packages/__docs__/src/Preview/index.tsx

```diff
@@ -48,14 +47,22 @@ class Preview extends Component<PreviewProps, PreviewState> {
     background: 'checkerboard'
   }
 
-  private _mountNode?: HTMLDivElement | null
-  private _root?: Root
+  static getDerivedStateFromProps(props: PreviewProps, state: PreviewState) {
+    if (props.error) {
+      return {
+        ...state,
+        error: props.error
+      }
+    }
+    return null
+  }
```

Comment:
If we get an error from the `PreviewErrorBoundary`, then map that error into the state.